crates/rolldown/tests/esbuild/ts/ts_enum_tree_shaking/artifacts.snap

```diff
@@ -23,6 +23,7 @@ var x = /* @__PURE__ */ function(x$1) {
 ```js
 
 //#region namespace-before.ts
+let x;
```

Comment:
@Dunqing can you double check this?

Reply:
Fixed in https://github.com/oxc-project/oxc/pull/10172?notification_referrer_id=NT_kwDOAcKkeLQxNTY0MDQwNTA5NToyOTUzMzMwNA#event-17086285809